src/Mod/TechDraw/App/DrawUtil.h

```diff
@@ -291,7 +280,7 @@ class TechDrawExport DrawUtil
 //template<> std::string DrawUtil::formatVector<Base::Vector3d>(const Base::Vector3d &v);
 
 // GCC BUG 85282, wanting this to be outside class body. This is only the declaration, the definition .cpp
-//template<> Base::Vector3d DrawUtil::toVector3d<QPointF>(const QPointF& v);
+//template<> Base::Vector3d DrawUtil::Base::convertTo<Base::Vector3d><QPointF>(const QPointF& v);
 
```

Comment:
This comment block can be removed

Reply:
There is another similar comment block above this one.
![image](https://github.com/user-attachments/assets/73e12ef8-628e-410d-99df-2db74b860ebb)
Should I remove this one too?

Reply:
My understanding of the one I suggested to remove was that it had been moved (and there was a bug in the code - `DrawUtil::Base`).

I don't know the same applies on the one you are referring to.

Reply:
> My understanding of the one I suggested to remove was that it had been moved (and there was a bug in the code - DrawUtil::Base).

Oh that was because I messed up the replacing and this one being a comment got missed.
I will just leave that second comment block as it is then.